bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java

```diff
@@ -44,7 +44,7 @@
   }
 
   public static Configuration configure(String projectId, String zoneName, String clusterName) {
-    Configuration config = new Configuration();
```

Comment:
This is not dataflow specific. Are you sure we want to disable loading default?
I don't know the difference, just want to make sure this change is OK.